tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/effects/debugger_effects_test.ts

```diff
@@ -1149,6 +1149,50 @@ describe('Debugger effects', () => {
       ]);
     });
 
+    for (const loadState of [DataLoadState.LOADED, DataLoadState.LOADING]) {
+      it('skips loading a loaded or loeading source file', () => {
```

Comment:
Let's use a test spec title that varies depending on the loadstate.  That way, one can read the Karma test results and know which one failed. e.g. perhaps something like
```
for (const spec of [{loadState: DataLoadState.Loaded, name: 'loaded'}, ...]) {
  it(`skips loading a source file that is ${spec.loadState}`, () => {
    // ...
  });
}
```

Reply:
Good point! Thanks for catching it.

Reply:
Done.